bundles/org.palladiosimulator.textual.tpcm/src/org/palladiosimulator/textual/tpcm/util/DefaultStoexTypeConverter.java

```diff
@@ -0,0 +1,36 @@
+package org.palladiosimulator.textual.tpcm.util;
+
+import java.util.Optional;
+
+import org.palladiosimulator.textual.tpcm.language.PrimitiveTypeEnum;
+
+import de.uka.ipd.sdq.stoex.analyser.visitors.TypeEnum;
+
+public class DefaultStoexTypeConverter implements IStoexTypeConverter {
+
+	@Override
+	public Optional<PrimitiveTypeEnum> getTPCMPrimitiveType(TypeEnum stoexEnum) {
```

Comment:
What about the STRING type?

Reply:
de.uka.ipd.sdq.stoex.analyser.visitors.TypeEnum does not have a String type.